test/verify/check-system-info

```diff
@@ -193,6 +193,12 @@ class TestSystemInfo(MachineCase):
         b.wait_in_text('#system_information_hostname_text', "Adventure Box (host1.cockpit.lan)")
         self.assertEqual(m.execute("hostname").strip(), "host1.cockpit.lan")
 
+        transient_hostname = m.image.split('-')[0]
+        m.execute("hostnamectl set-hostname ''")
+        b.wait_in_text('#system_information_hostname_text', transient_hostname)
+        # Allow error: Failed to create "/run/systemd/default-hostname": Permission denied
+        self.allow_journal_messages(".*scontext=system_u:system_r:systemd_hostnamed_t:s0 tcontext=system_u:object_r:init_var_run_t:s0 tclass=dir permissive=0")
```

Comment:
This is an actual OS bug, it even got reported and fixed in RHEL 9 a month ago here: https://bugzilla.redhat.com/show_bug.cgi?id=1966492 . It's not *quite* the same, as that concerned file removal, but it looks close enough. Can you please clone this for Fedora 34 (the little blue icon with two overlapping squares on the top right -- not that easy to see!) with mentioning the current systemd/selinux-policy package names?

We handle such unexpected messages in a slightly more delicate manner: On our bots project we maintain a [set of OS bugs labelled with "knownissue"](https://github.com/cockpit-project/bots/issues?q=is%3Aissue+is%3Aopen+label%3Aknownissue), e.g. https://github.com/cockpit-project/bots/issues/2162 for a recent SELinux bug. Then in https://github.com/cockpit-project/bots/tree/main/naughty we have a set of "failure patterns", like https://github.com/cockpit-project/bots/blob/main/naughty/fedora-34/2162-selinux-nft-pipe for the previous example (the file name is "issue number - short description"). Our test machinery tracks when and where it has acutally seen these issues, turns the "FAIL"s into a "SKIP" (i.e. a success with a warning), and makes a note in the knownissue.

Once it sees that a pattern is not used any more, it auto-files a PR like https://github.com/cockpit-project/bots/pull/2239 that cleans up the pattern -- and we can then close the downstream bug, or do another appropriate action.

So please: duplicate the bugzilla for  F34, create a bots knownissue for it, send a bots PR to add the pattern, and drop this line.

Reply:
I've filed a bugzilla issue, https://github.com/cockpit-project/bots/issues/2291 and created a pattern, I am just not 100% sure if it's correct.